ethapi/sfc_api.go

```diff
@@ -0,0 +1,304 @@
+	if err != nil {
+		return nil, err
+	}
+	return (*hexutil.Big)(v), err
+}
+
+// GetStakerPoI returns staker's PoI.
+func (s *PublicSfcAPI) GetStakerPoI(ctx context.Context, stakerID hexutil.Uint) (*hexutil.Big, error) {
+	v, err := s.b.GetStakerPoI(ctx, idx.StakerID(stakerID))
+	if err != nil {
+		return nil, err
+	}
+	return (*hexutil.Big)(v), err
+}
+
+// GetDowntime returns staker's Downtime.
+func (s *PublicSfcAPI) GetDowntime(ctx context.Context, stakerID hexutil.Uint) (map[string]interface{}, error) {
+	blocks, period, err := s.b.GetDowntime(ctx, idx.StakerID(stakerID))
+	if err != nil {
+		return nil, err
+	}
+	return map[string]interface{}{
+		"missedBlocks": hexutil.Uint64(blocks),
+		"downtime":     hexutil.Uint64(period),
+	}, nil
+}
+
+// GetDelegatorClaimedRewards returns sum of claimed rewards in past, by this delegator
+func (s *PublicSfcAPI) GetDelegatorClaimedRewards(ctx context.Context, addr common.Address) (*hexutil.Big, error) {
+	v, err := s.b.GetDelegatorClaimedRewards(ctx, addr)
+	if err != nil {
+		return nil, err
+	}
+	return (*hexutil.Big)(v), err
+}
+
+// GetStakerClaimedRewards returns sum of claimed rewards in past, by this staker
+func (s *PublicSfcAPI) GetStakerClaimedRewards(ctx context.Context, stakerID hexutil.Uint64) (*hexutil.Big, error) {
+	v, err := s.b.GetStakerClaimedRewards(ctx, idx.StakerID(stakerID))
+	if err != nil {
+		return nil, err
+	}
+	return (*hexutil.Big)(v), err
+}
+
+// GetStakerDelegatorsClaimedRewards returns sum of claimed rewards in past, by this delegators of this staker
+func (s *PublicSfcAPI) GetStakerDelegatorsClaimedRewards(ctx context.Context, stakerID hexutil.Uint64) (*hexutil.Big, error) {
+	v, err := s.b.GetStakerDelegatorsClaimedRewards(ctx, idx.StakerID(stakerID))
+	if err != nil {
+		return nil, err
+	}
+	return (*hexutil.Big)(v), err
+}
+
+// RPCMarshalStaker converts the given staker to the RPC output .
+func RPCMarshalStaker(it sfctype.SfcStakerAndID) map[string]interface{} {
+	return map[string]interface{}{
+		"id":               hexutil.Uint64(it.StakerID),
+		"totalStake":       (*hexutil.Big)(it.Staker.CalcTotalStake()),
+		"stake":            (*hexutil.Big)(it.Staker.StakeAmount),
+		"delegatedMe":      (*hexutil.Big)(it.Staker.DelegatedMe),
+		"isValidator":      it.Staker.IsValidator,
+		"isCheater":        it.Staker.IsCheater,
+		"address":          it.Staker.Address,
+		"createdEpoch":     hexutil.Uint64(it.Staker.CreatedEpoch),
+		"createdTime":      hexutil.Uint64(it.Staker.CreatedTime),
+		"deactivatedEpoch": hexutil.Uint64(it.Staker.DeactivatedEpoch),
+		"deactivatedTime":  hexutil.Uint64(it.Staker.DeactivatedTime),
+	}
+}
+
+func (s *PublicSfcAPI) addStakerMetricFields(ctx context.Context, res map[string]interface{}, stakerID idx.StakerID) (map[string]interface{}, error) {
+	blocks, period, err := s.b.GetDowntime(ctx, idx.StakerID(stakerID))
+	if err != nil {
+		return nil, err
+	}
+	res["missedBlocks"] = hexutil.Uint64(blocks)
```

Comment:
is it possible to introduce string constants like below?

MISSED_BLOCKS_FIELD="missedBlocks"
DOWNTIME_FIELD="downtime"
...